hive-udf/lib/src/main/java/com/featurebyte/hive/udf/BaseVectorAggregate.java

```diff
@@ -135,17 +126,28 @@ public void merge(AggregationBuffer agg, Object partial) {
       // Cast current aggregation buffer, and partial value.
       ListAggregationBuffer myagg = (ListAggregationBuffer) agg;
       List<Object> myList = (List<Object>) partial;
-      List<Object> container = myagg.container;
+      // Convert the parameter list to a list of doubles
+      List<Double> doubleList = new ArrayList<>();
+      for (Object o : myList) {
+        doubleList.add(Double.valueOf(o.toString()));
+      }
+      List<Double> container = myagg.container;
 
       // If there's no current value in the buffer, just set the partial value into the buffer.
       if (container == null || container.isEmpty()) {
-        myagg.container = myList;
+        myagg.container = doubleList;
         return;
       }
 
       // If not, compare the two lists, and update to the max value.
-      assert (container.size() == myList.size());
-      doMerge(container, myList);
+      if (container.size() != doubleList.size()) {
+        throw new RuntimeException(
```

Comment:
we update this to throw a runtimeexception as assert errors get swallowed unless a parameter is explicitly provided for when running the jar 

https://stackoverflow.com/a/56262724